docs/website/src/site/styles/layout/content.scss

```diff
@@ -1,11 +1,24 @@
-main {
-  background-color: var(--background-color);
-  margin-left: 260px;
-  padding: 36px 72px 0;
-  transition: background 300ms ease-in-out;
+@import './../breakpoints';
+
+.ds-container {
+  margin-left: 0;
+  padding: 9.6rem 3.2rem 0;
+  transform: translateX(0);
+  transition: transform 400ms, background 300ms ease-in-out, opacity 300ms;
   width: 100%;
 }
 
 .header-anchor {
+  color: var(--ds-color-raisin-black);
+  font-size: inherit;
   text-decoration: none;
 }
+
+@media screen and (min-width: $lg-breakpoint) {
```

Comment:
This is fine. In the future, perhaps we can switch to using custom media queries: https://www.stefanjudis.com/notes/can-we-have-custom-media-queries-please/